properties/P000016.md

```diff
@@ -10,4 +10,6 @@ refs:
 
 Every open cover of the space has a finite subcover.
 
+Equivalently, every open cover of the space has a finite refinement.
```

Comment:
Do you think we should say "has a finite open refinement"?  It's in fact true with finite refinements with any kind of sets, even sets that are not open and not closed.  But just wondering if people are more used to deal with open refinements, which fine too in this context.

Reply:
Some authors say "refinement" when they mean "open refinement". I'll add "[open]" since in this case it doesn't make a difference.

Reply:
```suggestion
Equivalently, every open cover of the space has an [open] finite refinement.
```